dll/win32/newdev/newdev.c

```diff
@@ -353,7 +353,10 @@ SearchDriverRecursive(
     BOOL retval = FALSE;
     HANDLE hFindFile = INVALID_HANDLE_VALUE;
 
-    wcscpy(DirPath, Path);
+    if (Path && Path[0])
+        wcscpy(DirPath, Path);
+    else
+        return retval;
```

Comment:
```suggestion
    if (!Path || !Path[0])
        return retval;

    wcscpy(DirPath, Path);
```

Reply:
Of course, mutch beter.
The suggested committee added changes to the header???

Reply:
> The suggested commit added changes to the header???

It seems so, for unknown reason.